docs/guide/advanced_under_development_features.rst

```diff
@@ -178,4 +178,80 @@ This dictionary should be passed in the ``setFromExcel`` and ``optimize`` functi
 
 Note that the time period would need to be adjusted to include the timesteps corresponding to 12 days (12 x 24 = 288 timesteps
 if hourly resolution is considered). Try the example on `selective days clustering <https://github.com/SPF-OST/optihood/blob/main/data/examples/selective_days_clustering.py>`_
-for a better grasp.
\ No newline at end of file
+for a better grasp.
+
+Ice storage
+------------
+
+The IceStorage class was implemented within the storages module of optihood. The formulation of the ice
+storage model is based on the solution of the energy conservation law applied to the water of the storage as
+per Carbonell et al. (2015). It is basically the same as the energy conservation law for hot water storage with
```

Comment:
Make sure the reference Carbonell et al. (2015) comes up at the end of the page

Reply:
done